package-lock.json

```diff
@@ -9511,14 +9511,14 @@
       "dependencies": {
         "file-type": {
           "version": "3.9.0",
-          "resolved": "https://registry.npmjs.org/file-type/-/file-type-3.9.0.tgz",
```

Comment:
Can we revert these changes to `package-lock.json`? They aren't relevant to this PR.